frontend/views/user-typeahead.html

```diff
@@ -0,0 +1,9 @@
+<a>
+  <div class="user-typeahead">
+    <div class="avatar avatar-64">
+      <img ng-src="{{match.model.avatar_url}}" width="64" />
+    </div>
+    <span bind-html-unsafe="match.label | typeaheadHighlight:query"></span>
```

Comment:
The module handles the escaping for you. This is what's recommended in their documentation. It's an unsafe bind because they add strong tags around the username.

It's worth noting that the highlighting only works on the username, not their full name. Not sure how to make that work.